rswag-api/lib/rswag/api/middleware.rb

```diff
@@ -29,6 +30,18 @@ def call(env)
 
       private
 
+      def parse_file(filename)
+        if /\.yml$/ === filename
```

Comment:
Two suggestions: 

- `filename.end_with?('.yml')` (if you only want .yml)
- `/\.ya?ml$/ === filename` (the [yaml faq](https://yaml.org/faq.html) suggests .yaml, so might be nice to support it)

Reply:
:+1:, sorry about the delay fixing this